src/App.vue

```diff
@@ -1,9 +1,18 @@
 <template>
   <div id="app">
+    <Navbar v-if="showNavbar" />
     <router-view></router-view>
   </div>
 </template>
 
 <script setup>
+import { computed } from 'vue'
+import { useRoute } from 'vue-router'
+import Navbar from '@/components/PollNavbar.vue'
 
+const route = useRoute()
+const showNavbar = computed(() => {
+  const noNavbarRoutes = ['PollLoginPage', 'SignupPage']
```

Comment:
this is the wrong method, can you not depend on user data?
if user is available show the navbar, else not @Shashank-002